tests/unit_tests/example_yaml/valid_auth_config.yaml

```diff
@@ -0,0 +1,11 @@
+stomp:
+  host: localhost
+  port: 61613
+oauth:
+  oidc_config_url: https://auth.example.com/realms/sample/.well-known/openid-configuration
```

Comment:
Is this technically valid if the url wouldn't actually work?

Reply:
I wrote that as a template happy to remove it as well